capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -156,4 +176,25 @@ public RedirectView saveEvent (
     this.eventRepository.save(event);
     return event.reviews;
   }
+
+  /**
+   * Filter out past events and events from other universities
+   * @param events list of events
+   * @param universityId user's university's datastoreId
+   * @return valid events
+   * * FILTER UNIVERSITY MANUALLY DUE TO SMALL EVENT VOLUME*
+   */
+  private List<Event> getValidEvents(Iterable<Event> events, long universityId) {
+    List<Event> validEvents = new ArrayList<Event>();
+    LocalDateTime now = LocalDateTime.now();
+    for (Event e : events) {
+      if (e.university.datastoreId == universityId) {
```

Comment:
should this be e.getUniversity().getDatastoreId()?

Reply:
Is that a best practice?